src/main/java/org/team1540/elmo/ejectors/EjectOuter.java

```diff
@@ -0,0 +1,22 @@
+package org.team1540.elmo.ejectors;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class EjectOuter extends SequentialCommandGroup {
+
+    BunnyEjectors.OuterEjector outerEjector;
+
+    public EjectOuter(BunnyEjectors.OuterEjector outerEjector) {
+        this.outerEjector = outerEjector;
+        addRequirements(outerEjector);
+
+        addCommands(
+                new InstantCommand(outerEjector::eject),
+                new WaitCommand(0.15), // wait for outer to start moving. // TODO: Test without waiting
+                new InstantCommand(()->outerEjector.setSuction(false))
```

Comment:
Looks like this might be switched with EjectInner, the outer scoring zone arm wants to release the tube and then knock it in.

Reply:
aha i mixed up the names D;